tests/Boost_Tests/main.cpp

```diff
@@ -0,0 +1,57 @@
+#define BOOST_TEST_DYN_LINK
```

Comment:
Since the header-only variant is included, is the dynamic linkage macro necessary?

Reply:
Thank you - I've removed it and will push soon...
(This was a hang-over from some example code that we copied)